### DISCUSSION TITLE
Show usage on github.com/codespaces

### DISCUSSION THREAD

**Discussion by reeceatkinson** [community/community#38780] [Codespaces] [answered]
### Select Topic Area

Product Feedback

### Body

It would be nice if you could see your usage/hours remaining on github.com/codespaces
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/78908925/201125038-4db36a1f-5788-41e6-9b9b-6cd87ef6cfd7.png">

**Comment by jaclynj** [comment, marked as answer]
Thanks for the feedback, I will pass that along to the codespaces product/ux folks. In the meantime you can see this info on the billing settings page like we do for other products. There you can see a breakdown of how much free usage you have consumed and how much is remaining. See [Viewing your GitHub Codespace Usage](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/viewing-your-github-codespaces-usage).

**Reply by reeceatkinson** [reply]
Thanks!

**Comment by spenserblack** [comment]
Great idea! I was about to start a very similar discussion, but I guess I'll put my thoughts here instead:

I've seen a lot of confusion about storage limits, lately (and I've been one of those confused people :rofl:)

I think it would be really helpful to see *projected* usage. Somewhere near "last used", I think it would be nice to see something like "uses 1GB per hour", to help identify why your storage usage is increasing. Also, apparently prebuilds count towards usage, so perhaps they should be listed in the Codespaces page somewhere? Once I found out they count towards usage, I had to hunt down the repos where I naively enabled prebuilds.
